pkg/config/config.go

```diff
@@ -207,9 +207,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("log_file", "")
 	config.BindEnvAndSetDefault("log_file_max_size", "10Mb")
 	config.BindEnvAndSetDefault("log_file_max_rolls", 1)
-	config.BindEnvAndSetDefault("log_level", "info")
+	config.BindEnvAndSetDefault("log_level", "info", "DD_LOG_LEVEL", "LOG_LEVEL")
 	config.BindEnvAndSetDefault("log_to_syslog", false)
-	config.BindEnvAndSetDefault("log_to_console", true)
+	config.BindEnvAndSetDefault("log_to_console", true, "DD_LOG_TO_CONSOLE", "DD_LOGS_STDOUT", "LOG_TO_CONSOLE")
```

Comment:
These new env vars need a corresponding change to the vars in config_template.yaml.

Reply:
I'd vote against supporting the `LOG_LEVEL` and `LOG_TO_CONSOLE` env vars on all agents. They don't follow our standard env var naming scheme, and I don't know of a compelling reason to make exceptions for them.

Since AFAIK these env vars were only supported by the process-agent so far, I think the risk of introducing a breaking change (i.e. having users that used these env vars successfully) by removing support for them is low. 

WDYT?

(this rationale may apply to `DD_LOGS_STDOUT` as well)

Reply:
I think we are open to removing `LOG_LEVEL` and `LOG_TO_CONSOLE`, I will go ahead and take these out and create a release note to match